src/App.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { Suspense, lazy } from "react";
+import { HashRouter, Route, Switch, Link } from 'react-router-dom';
+import HackerNews from "./components/HackerNews/HackerNews";
+
+
+export interface Info {
+    userInfo: any;
+    user: User;
+    isShow: boolean;
+    message?: string;
+}
+interface User {
+    Title: string;
+    ID: string;
+}
+interface IAppState {
+    info: any
+}
+export class App extends React.Component<{}, IAppState> {
+    private menus = {}
+    constructor(props) {
+        super(props);
+        this.state = {
+            info: undefined
+        }
+    }
+    render() {
+        return (
+            <HashRouter>
+                <div className="app" >
+                    <div className="sr-app-content">
+                        <Switch>
+                            <Route path="/" component={HackerNews} ></Route>
+                        </Switch>
+                    
+                    </div>
+
+                </div>
+            </HashRouter>)
+    }
+
+    componentDidMount() {
+        this.onInitial()
+    }
+
+    /**
+     * 初始化页面
+     */
+    async onInitial() {
+        let info: Info = {
```

Comment:
So what is been initialized here?

Reply:
this is for reading current user Information, but because of time limit, I didn't realize login function, So for now ,it didn't initiate user info, this App.tsx is just for routing function now